### ISSUE TITLE
Feat(Card): Add priority column

### ISSUE THREAD

**Issue by Us3r-gitHub** [tillywork/tillywork#154]
What if we add a new column `priority` to the card by default?

Currently, We can add it manually using custom fields, but I believe it should be included by default.

**Comment by MoShizzle** [comment]
I agree, one thing I'd like to explore is having default fields that don't need to be their own columns in the database (e.g card.data.priority instead of card.priority)

**Comment by Us3r-gitHub** [comment]
Is that what you mean? Are all fields custom fields?

**Comment by MoShizzle** [comment]
> Is that what you mean? Are all fields custom fields?

Currently no, but in the future yes I want all fields to be custom fields.
